gitbook/README.md

```diff
@@ -0,0 +1,124 @@
+[![Quality assurance](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/qualilty-assurance.yml/badge.svg?branch=master)](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/qualilty-assurance.yml)
+[![Code style](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/code-style.yml/badge.svg?branch=master)](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/code-style.yml)
+[![Maintainability](https://api.codeclimate.com/v1/badges/baa71ad27c3ce167cd7d/maintainability)](https://codeclimate.com/github/ExtraHorizon/javascript-sdk/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/baa71ad27c3ce167cd7d/test_coverage)](https://codeclimate.com/github/ExtraHorizon/javascript-sdk/test_coverage)
+
+# Extra Horizon JavaScript SDK
+
+This package serves as a JavaScript wrapper around all [Extra Horizon](https://www.extrahorizon.com/cloud-services) cloud services.
+
+## Features
+
+- [Auth][auth]: Provides authentication functionality. The Authentication service supports both OAuth 1.0a and OAuth 2.0 standards.
```

Comment:
On gitbook itself these links link to docs.extrahorizon.com, now they link to swagger. That seems like a mistake?